conda/recipes/rmm/meta.yaml

```diff
@@ -1,7 +1,9 @@
-# Copyright (c) 2019, NVIDIA CORPORATION.
+# Copyright (c) 2019-2020, NVIDIA CORPORATION.
 
 {% set version = environ.get('GIT_DESCRIBE_TAG', '0.0.0.dev').lstrip('v') + environ.get('VERSION_SUFFIX', '') %}
 {% set py_version=environ.get('CONDA_PY', 35) %}
+{% set cuda_version='.'.join(environ.get('CUDA_VERSION', '10.1').split('.')[:2]) %}
```

Comment:
Is this hard-coding 10.1?

Reply:
No, this reads the environment variable `CUDA_VERSION` into a jinja variable which can be used in jinja expressions below

Reply:
So what is the 10.1?

Reply:
> So what is the 10.1?

The default if the environment variable doesn't exist.